From: Dave Jiang <dave.jiang@intel.com>
Date: Mon, 15 May 2023 09:07:19 -0700
Subject: ACPICA: Fix misspelled CDAT DSMAS define
Patch-mainline: v6.6-rc1
Git-commit: 6399878f4cb4cc7440965dbd679ee650c59b19e3
References: jsc#PED-6747

ACPICA commit 32a50922b66a9e288b9a9b4740de86a542668a43

ACPI_CEDT_DSMAS_NON_VOLATILE -> ACPI_CDAT_DSMAS_NON_VOLATILE

Link: https://github.com/acpica/acpica/commit/32a50922
Signed-off-by: Dave Jiang <dave.jiang@intel.com>
Signed-off-by: Bob Moore <robert.moore@intel.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 include/acpi/actbl1.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/acpi/actbl1.h
+++ b/include/acpi/actbl1.h
@@ -402,7 +402,7 @@ struct acpi_cdat_dsmas {
 
 /* Flags for subtable above */
 
-#define ACPI_CEDT_DSMAS_NON_VOLATILE        (1 << 2)
+#define ACPI_CDAT_DSMAS_NON_VOLATILE        (1 << 2)
 
 /* Subtable 1: Device scoped Latency and Bandwidth Information Structure (DSLBIS) */
 
